### DISCUSSION TITLE
[FR] Allow custom root domain option for hosts file (Pi-hole)

### DISCUSSION THREAD

**Discussion by jtovar43** [pivpn/pivpn#1454] [Feature requests]
**Describe the solution you'd like**
It would be nice to have a way to specify a root domain name to use in the hosts file referenced in #942 instead of the current `.pivpn` domain. This could be added as a environment variable and referenced in `scripts/wireguard/makeCONF.sh` instead of statically appending `.pivpn`

**Describe alternatives you've considered**
Currently, you have to manually edit the `/etc/pivpn/hosts.wireguard` file to change the root domain for each client individually.

**Comment by coolapso** [comment]
Moving to discussions.
